### ISSUE TITLE
Assertion triggered when using libdatachannel

### ISSUE THREAD

**Issue by dtzxporter** [paullouisageneau/libjuice#163]
```
Assertion failed: (impl->map_count >= count), function remove_map_entries, file conn_mux.c, line 130.
```

Happens randomly, hard to track down right now.

**Comment by paullouisageneau** [comment]
Thank you for reporting. It appears that there is a mistake in `insert_map_entry()` resulting in `map_count` not being always incremented as it should, possibly leading to this error.

It should be fixed by https://github.com/paullouisageneau/libjuice/pull/164

**Comment by paullouisageneau** [comment]
Fix released in `v1.0.3`.

**Comment by paullouisageneau** [comment]
Fix released in `v1.0.3`
